EKrut-Client/src/ekrut/client/managers/ClientTicketManager.java

```diff
@@ -0,0 +1,87 @@
+package ekrut.client.managers;
+
+import java.util.ArrayList;
+
+import ekrut.entity.Ticket;
+import ekrut.net.ResultType;
+import ekrut.net.TicketRequest;
+import ekrut.net.TicketRequestType;
+import ekrut.net.TicketResponse;
+import ekrut.net.UserRequest;
+import ekrut.net.UserResponse;
+
+//need to add- sendRequest
+
+
+/**
+ * The ClientTicketManager class is responsible for managing tickets for a client.
+ * This includes updating the status of a ticket and fetching tickets.
+ * 
+ * @author Noy Malka
+ */
+
+public class ClientTicketManager {
+
+	
+	/**
+     * Updates the status of a given ticket.
+     * 
+     * @param ticket the ticket to update
+     * @return the result of the update operation
+     * @throws NullPointerException if ticket is null
+     * @throws RuntimeException if there is an error updating the ticket's status
+     */
+	
+	public ResultType updateTicketStatus(Ticket ticket) throws Exception{
+		if (ticket==null) {
+			throw new IllegalArgumentException("provided null ticket");
+		}
+		
+		TicketRequest ticketRequest = new TicketRequest(TicketRequestType.UPDATE_STATUS,ticket.getTicketId()); 
+		TicketResponse ticketResponse = sendRequest(ticketRequest);
+		if (ticketResponse.getResultType().toString()!="OK") {
+			throw new RuntimeException("Error updating ticket's status"); //**Need to think whats better then Runtime here

```

Comment:
This is not how you compare strings. `strA.equals(strB);`
You shouldn't use strings here anyway just compare the enums: `ticketResponse.getResultType() != ResultType.OK`
Same thing happens in `fetchTickets()`.

Also, you probably shouldn't throw an exception here, it's better to just return the result. Otherwise, there would be no point in the method returning `ResultType` because it can only ever return `OK` so a void return type and no exception would indicate the operation was successful.